scripts/MultiPlotting/multi_plotting_context.py

```diff
@@ -30,20 +31,36 @@ def addSubplot(self, name, subplot):
 
     def addLine(self, subplotName, workspace, specNum):
         try:
-            if len(workspace) > 1:
+            if isinstance(workspace, str):
+                ws = mantid.AnalysisDataService.retrieve(workspace)
+                self.subplots[subplotName].addLine(ws, specNum)
+
+            elif len(workspace) > 1:
                 self.subplots[subplotName].addLine(
                     workspace.OutputWorkspace, specNum)
             else:
                 self.subplots[subplotName].addLine(workspace, specNum)
         except:
-            print("cannot plot workspace")
+            print("cannot plot workspace", workspace)
```

Comment:
Shouldn't this error rather go to the logger? In any case, we should use Python 3 compatible printing: `print('cannot plot workspace {}'.format(workspace))`